### ISSUE TITLE
Multiple elements with the Navigation role do not have unique labels

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#5504]
Per WCAG 2.1 navigation elements (such as the navbar for scenes, or the nav element used for sheet tabs and sidebar tabs) each navigation element should have an explicit associated label or ARIA label.
